Source/Views/AlertView.swift

```diff
@@ -158,8 +158,8 @@ final class AlertView: UIView, AlertControllerViewRepresentable {
             self.messageLabel.leftAnchor.constraint(equalTo: self.leftAnchor, constant: contentPadding.left),
             self.messageLabel.rightAnchor.constraint(equalTo: self.rightAnchor,
                                         constant: -contentPadding.right),
-            self.messageLabel.firstBaselineAnchor.constraint(equalTo: self.titleLabel.lastBaselineAnchor,
-                                                            constant: self.visualStyle.verticalElementSpacing)
+            self.messageLabel.topAnchor.constraint(equalTo: self.titleLabel.bottomAnchor,
```

Comment:
This is the key part of the change.  Using the baseline anchors means that the font descenders can overlap with the next line if the verticalElementSpacing constant is not large enough for large fonts.  Using top and bottom anchors means they will never overlap.  I found that a constant of 0 looks good, because the fonts natural spacing forms the boundary between the two elements.

Reply:
Interesting. Do you have an example of an alert where that happens? I used the baselines because that's what UIAlertController uses and I'm wondering how Apple circumvents that problem so I'd like to play around with it a bit.

Reply:
I'm curious, how do you know how Apple's source code handles this?  This picture is from SDCAlertView before my changes.
![screen shot 2017-11-15 at 12 52 43 pm](https://user-images.githubusercontent.com/829682/34320324-eef7fcdc-e7b4-11e7-8fb7-ff5a24c6260f.png)
Here's a picture after the changes:
![screen shot 2017-11-15 at 12 51 21 pm](https://user-images.githubusercontent.com/829682/34320332-2b9ef190-e7b5-11e7-869e-3945f794d165.png)

Reply:
In this particular case, I present a regular `UIAlertController` and use the view debugger to poke around a bit. For other cases I've also done some different reverse engineering but that wasn't necessary here. See comment below for what I think we should do to fix this.

Reply:
I think my approach works well, makes sense and has the benefit of being simple.  FWIW: I don't think you should base your implementation off of what you can glean from a view debugger.  Apple could have tons of special code to present the alertview differently based on circumstances.  Also, having worked at Apple for 4 years, there's nothing sacrosanct about Apple's code, you need to solve the problem based on what the BEST solution is, not try to copy them.  But If you find a solution that works for the dynamic type in all of the sizes that works better than mine, is simpler than mine, or you like it better than mine, than you should obviously use it.

Reply:
Nah I don’t use Apple’s code as an example of great software examples, but when I first built the library it was a fun challenge to build something that 100% matched Apple’s alert U. With a reference implementation at hand the most obvious solution is then to just build it the same way.

But over the years Apple has added so much stuff to the alert UI that that’s not really realistic anymore, so it becomes a balance of how the UI can stay as true to the original as possible without doing the crazy stuff they do. I agree that in this case it’s probably OK to go with your solution and call it a day.